src/components/landing/OfferingCard.js

```diff
@@ -0,0 +1,25 @@
+import { FaUserAlt } from "react-icons/fa";
+
+const OfferingCard = ({ title, description, icon }) => {
+  if (!icon && !title && !description) {
+    return <div></div>;
+  }
+  if (!title) {
+    title = "Title";
+  }
+  if (!icon) {
+    icon = <FaUserAlt className="size-12 text-[#4F9AD7]" />;
+  }
+  if (!description) {
+    description = "Description";
+  }
+  return (
+    <div className="h-80 w-72 rounded-3xl p-8 bg-[#AEC6D8] flex flex-col justify-between">
```

Comment:
Don't use raw hex codes in the color, should be added as a custom color in the config